src/Database/Log/DebugLog.php

```diff
@@ -135,15 +135,16 @@ public function log($level, string|Stringable $message, array $context = []): vo
 
         // This specific to Elastic Search
         if (!$query instanceof LoggedQuery && isset($context['request']) && isset($context['response'])) {
-            $this->_totalTime += $context['response']['took'];
+            $took = $context['response']['took'] ?? 0;
+            $this->_totalTime += $took;
 
             $this->_queries[] = [
                 'query' => json_encode([
                     'method' => $context['request']['method'],
                     'path' => $context['request']['path'],
                     'data' => $context['request']['data'],
                 ], JSON_PRETTY_PRINT),
-                'took' => $context['response']['took'] ?: 0,
+                'took' => 0,
```

Comment:
```suggestion
                'took' => $ook,
```

This should still be set, right?